internal/goobj/read.go

```diff
@@ -679,8 +679,6 @@ func (r *objReader) parseObject(prefix []byte) error {
 			f.FuncData = make([]FuncData, r.readInt())
 			for i := range f.FuncData {
 				f.FuncData[i].Sym = r.readSymID()
-			}
-			for i := range f.FuncData {
 				f.FuncData[i].Offset = int64(r.readInt()) // TODO
```

Comment:
I'm not very familiar with this project, but doesn't this change the behavior? The order was previously like `r.readSymID(), r.readSymID(), r.readSymID(), r.readInt(), r.readInt(), r.readInt()`, but now it'll be `r.readSymID(), r.readInt(), r.readSymID(), r.readInt(), r.readSymID(), r.readInt()`.